src/lem/store.rs

```diff
@@ -1109,6 +1109,10 @@ impl Ptr {
         }
     }
 
+    pub fn fmt_to_string_dammit<F: LurkField>(&self, store: &Store<F>) -> String {
```

Comment:
We can rename it to `fmt_to_string_simple`

Reply:
Done.